libbebop/bebop.h

```diff
@@ -283,6 +301,9 @@ class Bebop
     LimitValues<meters_per_second_t> m_VerticalSpeedLimits;
     LimitValues<degrees_per_second_t> m_YawSpeedLimits;
     std::atomic<unsigned char> m_BatteryLevel;
+    std::atomic<RelativeMoveState> m_RelativeMoveState{ RelativeMoveState::Success };
```

Comment:
Is there a reason this isn't initialised in the err 'initial' state? :smile:

Reply:
Hmm it doesn't *look* like there's a good reason... but if there was, I wrote this so long ago I wouldn't remember anyway :wink: . I'll fix this up